gamestonk_terminal/cryptocurrency/binance_model.py

```diff
@@ -0,0 +1,253 @@
+        "12hour": client.KLINE_INTERVAL_12HOUR,
+        "1week": client.KLINE_INTERVAL_1WEEK,
+        "1min": client.KLINE_INTERVAL_1MINUTE,
+        "3min": client.KLINE_INTERVAL_3MINUTE,
+        "5min": client.KLINE_INTERVAL_5MINUTE,
+        "15min": client.KLINE_INTERVAL_15MINUTE,
+        "30min": client.KLINE_INTERVAL_30MINUTE,
+        "1month": client.KLINE_INTERVAL_1MONTH,
+    }
+    interval_choices = list(interval_map.keys())
+
+    parser = argparse.ArgumentParser(
+        prog="candle",
+        add_help=False,
+        description="Program to plot candles for binance data",
+    )
+    parser.add_argument(
+        "-i",
+        "--interval",
+        help="Interval to get data",
+        choices=interval_choices,
+        dest="interval",
+        default="1day",
+        type=str,
+    )
+
+    parser.add_argument(
+        "-l", "--limit", dest="limit", default=100, help="Number to get", type=int
+    )
+
+    try:
+        ns_parser = parse_known_args_and_warn(parser, other_args)
+        if not ns_parser:
+            return
+        candles = client.get_klines(
+            symbol=coin,
+            interval=interval_map[ns_parser.interval],
+            limit=ns_parser.limit,
+        )
+
+        # Response
+        # 1499040000000, // Open time
+        # "0.01634790", // Open
+        # "0.80000000", // High
+        # "0.01575800", // Low
+        # "0.01577100", // Close
+        # "148976.11427815", // Volume
+        # 1499644799999, // Close time
+        # "2434.19055334", // Quote asset volume
+        # 308, // Number of trades
+        # "1756.87402397", // Taker buy base asset volume
+        # "28.46694368", // Taker buy quote asset volume
+        # "17928899.62484339" // Ignore.
+
+        candles_df = pd.DataFrame(candles).astype(float).iloc[:, :7]
+
+        candles_df.columns = [
+            "Time0",
+            "Open",
+            "High",
+            "Low",
+            "Close",
+            "Volume",
+            "Time1",
+        ]
+        candles_df.index = pd.to_datetime(
+            (candles_df["Time0"] + candles_df["Time1"]) / 2, unit="ms"
+        )
+
+        plot_candles(
+            candles_df,
+            f"{coin} from {candles_df.index[0].strftime('%Y/%m/%d')} to "
+            f"{candles_df.index[-1].strftime('%Y/%m/%d')}",
+        )
+
+    except Exception as e:
+        print(e)
+        print("")
+
+
+def balance(coin: str):
+    """
+    Get account holdings for asset
+    Parameters
+    ----------
+    coin: str
+        Coin to get holdings of
+
+    Returns
+    -------
+
+    """
+    client = Client(BINANCE_API_KEY, BINANCE_SECRET_KEY)
+    try:
+        current_balance = client.get_asset_balance(asset=coin)
+        if current_balance is None:
+            print("Check loaded coin")
+            return
+        print("")
+        print(
+            f"You currently have {current_balance['free']} free and {current_balance['locked']} locked"
```

Comment:
Can we round this somehow? 
```
You currently have 0.00000000 free and 0.00000000 locked
```
In case of bitcoin for instance we would want more decimal places. We can do something smart like multiply the coins by the current price, and see how many decimal places we need so that the value is under 1. That is the number of decimal places we would round to. What you think?

Also, I recommend something like:
```
FREE       : 15  (75%)
LOCKED : 5    (25%)
```
It's more nice visually, and you can add that percentage

Reply:
that didn't type set well, but I could do a tabulate table (also honestly I don't know how much someone would have locked...should be small right?

Reply:
I'm not sure what this lock even means to be fully honest. 

I stacked DOT the other day, not sure if that's considered "locked". But in my case I stacked all my coins, so if that's considered "locked" then I think it can be a big amount.

Reply:
So "locked" may be staked (is that what you mean by stacked...)?

Reply:
Yes, it is indeed!

Reply:
So I haven't changed anything for rounding.  As you say it depends on the coin price, but I think crypto is just inherently messy.  Honestly this is also a command that is probably run very rarely (since I am not doing prices -- which I could) so you probably want to know your entire holdings when you run it.  (Should I add price?)